runtime/auth/aws-signing-common/common/src/aws/smithy/kotlin/runtime/auth/awssigning/middleware/AwsSigningMiddleware.kt

```diff
@@ -29,6 +29,8 @@ public class AwsSigningMiddleware(private val config: Config) : ModifyRequestMid
             requireNotNull(config.signer) { "A signer must be specified for the middleware" }
             return AwsSigningMiddleware(config)
         }
+
+        public const val AWS_CHUNKED_THRESHOLD: Int = CHUNK_SIZE_BYTES * 16
```

Comment:
fix: Documentation

question: Should this be `public` or `internal`? (if public should it be `@InternalApi`?)

Reply:
Currently it needs to be `internal` to be accessible in some test cases. Since the class AwsSigningMiddleware is `@InternalApi`, does its companion object need to be marked, or is it inherited?